cmd/commom/commom.go

```diff
@@ -79,3 +85,66 @@ func MatchPath(path, pattern string) bool {
 	}
 	return match
 }
+
+func ConfigureTracer(ctx context.Context, serviceName string) (func() error, error) {
+	tracerUrl := os.Getenv("OTEL_EXPORTER_JAEGER_ENDPOINT")
+	switch {
+	case tracerUrl != "":
+		return configureJaeger(ctx, serviceName)
+	default:
+		return configureStdout(ctx, serviceName)
+	}
```

Comment:
```suggestion
	tracerType := config.GetString("api.tracing.type")
	switch {
	case tracerType == "jaeger":
		return configureJaeger(ctx, serviceName)
	default:
		return configureStdout(ctx, serviceName)
	}
```

I think this leaves things clearer for the user than an environment var that is specific to the lib. 
Let the jaeger dependency handle its configuration instead

Reply:
I was thinking of it, and maybe logging traces to STDOUT isn't a good idea.
Thus, we may panic if the tracing is enabled, and we don't have the tracing url set, wdyt?

Reply:
Another thing, is that we should make explicit that we only accept Jaeger, so the configuration will be something like this:
```yaml
api:
  ...
  tracing:
    jaeger:
      enabled: true
      url: "http://jaeger.url/api/traces"
```
Does this make sense?

Reply:
I like this, this is a better idea, the point is just to make it clearer to the user how to enable/disable the jaeger with the configuration